bson/src/main/scala/DefaultBSONHandlers.scala

```diff
@@ -120,24 +120,6 @@ trait DefaultBSONHandlers {
   implicit def bsonBooleanLikeReader[B <: BSONValue] =
     new BSONBooleanLikeReader[B]
 
-  // Collections Handlers
```

Comment:
I don't see the goal, those instances are not low priority for me

Reply:
I can prepare some examples with "-Xlog-implicits" so you can see the compiler trying to use BSONArrayCollectionWriter where it is not needed, but like I said above other libraries to it like this: [play-json](https://github.com/playframework/play-json/blob/master/play-json/shared/src/main/scala/play/api/libs/json/Reads.scala#L186) and [circe](https://github.com/circe/circe/blob/master/modules/core/shared/src/main/scala/io/circe/Decoder.scala#L281)

Reply:
There are other reason why Play JSON is doing it in this way (I'm contrib on both).

Reply:
I will prepare the examples then. Maybe later today

Reply:
thx